single_cell_RNAseq/modules/pipeline_tasks.nf

```diff
@@ -441,31 +465,11 @@ process UNMERGE_FMX {
 process SEPARATE_FMX {
    publishDir "${params.project_dir}/data/single_cell_GEX/processed/${library}/freemuxlet", mode: 'copy', pattern: "${library}*"
    publishDir "${params.project_dir}/data/single_cell_GEX/logs/${library}/", mode: 'copy', pattern: ".command.log", saveAs: { filename -> "separate_fmx.log" }
-  input:
-   tuple val(library), path(library_files)
-
-  output:
-   tuple path("${library}.clust1.samples.gz"), path("${library}.clust1.vcf.gz"), path("${library}.lmix"), emit: fmx_files
-   tuple val(library), path("${library}.clust1.samples.reduced.tsv"), emit: sample_map
-   path(".command.log"), emit: log
-
-  """
-  gunzip -f ${library}.clust1.samples.gz
-  awk {'printf (\"%s\t%s\t%s\t%s\t%s\\n\", \$2, \$3, \$4, \$5, \$6)'} ${library}.clust1.samples > ${library}.clust1.samples.reduced.tsv
-  gzip -f ${library}.clust1.samples
-  """
-}
-
-// TODO: unify the two processes
-process SEPARATE_FMX_PRE {
```

Comment:
I'm happy you unified these processes!

I remember now why I duplicated it as a short term solution. There are other pipelines that reference `SEPARATE_FMX` and I wanted to make sure I didnt break them by being explicit about the input:

`tuple val(library), path(vcf_file), path(sample_file), path(lmix_file)`

Might be worth running those pipelines to be sure?